src/components/Avatar/Avatar.jsx

```diff
@@ -95,26 +102,62 @@ const Avatar = ({
     return badges.length > 0 ? <div className={cx(bemHelper({ element: "badges" }))}>{badges}</div> : null;
   }, [size, topLeftBadgeProps, topRightBadgeProps, bottomLeftBadgeProps, bottomRightBadgeProps]);
 
+  const TooltipContainer = ({ children }) => {
```

Comment:
if the issue is that you have 2 conditions it will be still clearer and more efficient to write to conditions instead creating 2 anonymous components in the render function :)